.rubocop.yml

```diff
@@ -55,3 +52,18 @@ Style/StringLiterals:
 
 Style/WordArray:
   EnforcedStyle: brackets
+
+Style/StringLiterals:
```

Comment:
:no_good_man: I'm a double-quotes guy but more than that, I'm a don't-hassle-contributors-with-quote-drama guy